spaces/domains.json

```
@@ -177,5 +177,7 @@
   "governance.treedefi.com" : "treedefi.eth",
   "vote.m1guelpf.me": "token.m1guelpf.eth",
   "gov.parallaxprotocol.com": "parallaxprotocol.eth",
-  "snapshot.everipedia.com": "everipediaiq.eth"
+  "snapshot.everipedia.com": "everipediaiq.eth",
+  "vote.nerve.fi": "nervefinance.eth"
+
```

Please remove the empty line. Thank you.
```suggestion
  "vote.nerve.fi": "nervefinance.eth"
```